compassion_norway_payment/wizards/load_mandate_wizard.py

```diff
@@ -84,10 +86,11 @@ def generate_new_mandate(self):
                             else:
                                 mandate_id = valid.id
                         data_dict = {"name_file": wizard.name_file, 'mandate_id': mandate_id,
-                                     'old_mandate_state': old_state, 'is_cancelled': is_cancelled}
+                                     'old_mandate_state': old_state, 'is_cancelled': is_cancelled,
+                                     'kid': transaction.kid}
                         if data_dict['mandate_id'] not in data:
                             data.append(data_dict)
             self._log_results(data)
-            return self.load_views
+            return self.load_views, self.unlink()
```

Comment:
Instead of manually unlinking the records, I would prefer that the class inherits models.TransientModel instead of models.Model. I fear that destroying them here can lead to issues.